### DISCUSSION TITLE
The plugin calling DALL-E-3 is unstable, and the errors cannot be eliminated

### DISCUSSION THREAD

**Discussion by leoterry-ulrica** [danny-avila/LibreChat#1399] [Troubleshooting]
### What happened?

When calling the DALL-E-3 plugin, sometimes it can be accessed, and sometimes it cannot. Moreover, once an error occurs, when trying to make it draw again, it consistently returns the invalid image address from the last error.

### Steps to Reproduce

1、select DALL-E-3 plugin
2、prompt：draw a white cat
3、once exception，i cannot try again~

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
No issues here. You likely are using a reverse proxy

**Comment by leoterry-ulrica** [comment]
> No issues here. You likely are using a reverse proxy

Can't I use the reverse proxy ？

**Reply by fuegovic** [reply]
Yes you can, but reverse proxies are less stable than the official API
